src/BloomExe/MiscUI/ProblemReporterDialog.cs

```diff
@@ -332,6 +332,28 @@ private bool SubmitToYouTrack()
 							zip.Save();
 							AddAttachment(bookZip.Path);
 						}
+						catch (InvalidRequestException e)
+						{
+							// We get this rather unspecific exception (with the even more unhelpful message 'NoContent') if the attachment is too large.
+							// There might of course be other reasons.
+							bookZip.Detach();
```

Comment:
Yes, I an I know why the case *I* was handling needed it. But *your* case doesn't :-)